src/lib/effect.js

```diff
@@ -0,0 +1,27 @@
+let counter = 0;
+
+export default class Effect {
```

Comment:
A test file for this class (`effect.spec.js`), would be trivial to write (just check that the layer symbol imports, that you can construct the class and that you can call `setNeedsRedraw`). 

It will make your PR look much more complete.